src/components/pages/Admin/BacklogStoryCard.js

```diff
@@ -13,7 +13,7 @@ const BacklogStoryCard = ({ story }) => {
   } = story;
   const date = new Date(timeSubmitted);
 
-  const [statusColor, setStatusColor] = useState(
+  const [statusColor] = useState(
```

Comment:
Again...if it works I guess that is cool, but I don't think you need statusColor to be stored in state.  It could just be a function or the conditional rendering could be directly applied in the color attribute of the element.